### PR TITLE
Simplify pspawner example

### PR DESCRIPTION
**Pull request by blacksound**
Please forgive me if this is an overstep but I have a suggestion for simplifying example 3 in the Pspawner help file.

Based on my personal experience the original example is a bit to complex for illustrating the point the example tries to make.
I have taken the liberty of modifying it in order to make the point a bit (in my opinion) more succinct, and  added some comments.
I hope I have managed to keep most of the nuances from the original example.

This simplification is by no means an improvement in any musical sense.

### CODE REVIEW

File: HelpSource/Classes/Pspawner.schelp [hunk 1]
```diff
@@ -105,37 +105,36 @@ Pspawner({ | sp |
...
+	Pbind(*[mtranspose: Pkey(\mtranspose) + Pstutter(8, Prand([0,[0,3,-2],[0,2,4]], inf)) ] ), //make some notes into triads
+	Pbind(*[ctranspose: Pwhite(-0.1, 0.1)]), //add random detuning to notes
+	Pbind(*[\scale, Scale.minor]), //alter the scale
+	Pn(Pseq([p, (type:\rest, dur: 1.0)]) ), //repeat the pattern after a 1 second pause
 	Pbind(*[
-		db: Pstep(Pseq([-10, -13, -13, -11, -13, -13], inf), 0.1) - 10,
-		mtranspose: Pstep(Pwhite(-7, 7), Prand([5,4,2],inf) )
- 	])
- ).play(protoEvent: Event.default)
+	 db: Pstep(Pseq([-10, -13, -13, -11, -13, -13], inf), 0.1) - 10,
```
Comment (crucialfelix) [inline HelpSource/Classes/Pspawner.schelp]: is that a mixed tab and space ?

**Comment by crucialfelix** [comment]
Simpler examples are always better IMO. Its not supposed to be a riddle with 10 ideas to learn.

**Comment by jamshark70** [comment]
I might also suggest replacing the `*[name: pattern]` syntax with the more canonical `\name, pattern`. Ron has a good reason to use the alternate syntax in his classes, but using it casually in a help file with no explanation is a handy way to confuse new users.

**Comment by crucialfelix** [comment]
I agree. Even I didn't know what the hell that meant: `*[name: pattern]

I know (key: value) but I don't know what [key: value] makes I don't want to have 10 ways of using the language. It doesn't help, it hinders. my 2c

**Comment by blacksound** [comment]
I think changing from `*[name, pattern]` to `\name pattern` is not something that should be done in this pull request.
This leads to a general discussion whether there should be a consistent code style throughout the documentation. It is better to have that discussion on sc-dev in my opinion.

**Pull request merged**